generator/src/main/java/com/scottlogic/deg/generator/validators/GenerationConfigValidator.java

```diff
@@ -70,14 +70,14 @@ private void checkGenerateOutputTarget(ArrayList<String> errorMessages,
     private void checkViolationGenerateOutputTarget(ArrayList<String> errorMessages,
                                                     OutputTarget outputTarget, int ruleCount) {
         if (outputTarget instanceof FileOutputTarget) {
```

Comment:
You could exit early if `outputTarget` isn't a `FileOutputTarget` which could save some of the repeated hard-casts

Reply:
I would rather not introduce changes here as this is being worked on in another ticket (not sure which at the moment)